components/sensor/max31865.rst

```diff
@@ -0,0 +1,70 @@
+MAX31865 Platinum RTD Temperature Sensor
+========================================
+
+.. seo::
+    :description: Instructions for setting up MAX31865 platinum RTD temperature sensors.
+    :image: max31865.jpg
+
+The ``max31865`` temperature sensor allows you to use your max31865 RTD
+temperature sensor (`datasheet <https://datasheets.maximintegrated.com/en/ds/MAX31865.pdf>`__) with ESPHome
+
+.. figure:: images/max31865-full.jpg
+    :align: center
+    :width: 50.0%
+
+    MAX31865 Sensor. Image by `Adafruit`_
+
+.. _Adafruit: https://www.adafruit.com/product/3328
+
+As the communication with the MAX31865 is done using SPI, you need
+to have an :ref:`spi bus <spi>` in your configuration with both **miso_pin** and **mosi_pin** set.
+
+- ``VIN`` connects to 5V (``3V3`` will output 3.3V), or directly connect ``3V3`` to 3.3V
+- ``GND`` connects to ground
+- ``CLK`` connects to the SPI **clk_pin**
+- ``SDO`` connects to the SPI **miso_pin**
+- ``SDI`` connects to the SPI **mosi_pin**
+- ``CS`` connects to a free GPIO pin
+- ``RDY`` is not used by this library
```

Comment:
```suggestion
- ``RDY`` is not used by ESPHome
```

ESPHome is strictly speaking not a library, but a framework.

Reply:
Understood. I will implement your suggestions this evening.